src/stores/VideoChannelStore.ts

```diff
@@ -154,6 +160,7 @@ export default class VideoChannelStore extends AsyncStoreWithClient<null> {
         // Participant data and mute state will come down the event pipeline quickly, so prepare in advance
         this.activeChannel = messaging;
         this.roomId = roomId;
+        const room = this.room;
```

Comment:
somewhat awkward that we define the variable all the way up here then 45 lines later use it for the first time.

Also, this doesn't seem to need to be a variable?